src/Components/ToDoItem/ToDoItem.tsx

```diff
@@ -10,10 +15,57 @@ const TodoItem: React.FC<TodoItemProps> = ({ todo }: TodoItemProps) => {
   // const { t } = useTranslation();
 
   return (
-    <div>
-      <p>{todo.title}</p>
+    <div className={css(styles.taskWrapp)}>
+      <div className={css(styles.taskHeader)}>
+        <p className={css(typography.bigFont, styles.taskTitle)}>
+          {todo.title}
+        </p>
+        <Icon iconName={Icons.faGraduationCap} iconSize="2x" />
+      </div>
+      <p className={css(typography.normalFont, styles.taskDescription)}>
+        Description:
+      </p>
+      <p className={css(styles.taskDescriptionText)}>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt
+        aliquam accusamus enim quae iure blanditiis labore quisquam inventore
+        reiciendis voluptate. Numquam vero consequuntur saepe nihil.
+      </p>
     </div>
   );
 };
 
+const styles = StyleSheet.create({
+  taskWrapp: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-around",
+    width: "370px",
+    height: "220px",
+    margin: "10px 5px",
+    padding: "6px",
+    backgroundColor: `${palette.blue}`,
+    border: `2px solid ${palette.darkBlue}`,
+    borderRadius: "5px 5px 5px 5px",
+    color: `${palette.white}`,
+    boxShadow: `2px 2px 4px ${palette.darkGray}`,
```

Comment:
If you use shadow in many places and it is look the same you should move these shadow styles into typography.ts / styles.ts